src/test/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/ConfigurationAsCodeTest.java

```diff
@@ -0,0 +1,55 @@
+package com.sonymobile.jenkins.plugins.mq.mqnotifier;
```

Comment:
Shouldn't there be a copyright header in this file?

Reply:
Correct, I forgot about that. I'll add one.